### PR TITLE
[PLATFORM-981] Force the default possible value do display “correctly”

### PR DESCRIPTION
**Pull request by mondoreale**
What do you think @timoxley?

The `optionMap.get(value) || value` really isn't perfect. It seems to work tho. And I think we should address the issue at its core (e&e). This should do it for now tho.

Please try to break it.

### CODE REVIEW

File: app/src/editor/canvas/components/Ports/Value/Select/index.jsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ const Select = ({
                 {/* `select` holding a currently selected value. This hidden (`visibility: hidden`) control
                     dictates the width of the actual (visible) control above. */}
                 <select className={styles.spaceholder}>
-                    <option>{optionMap.get(value)}</option>
+                    <option>{optionMap.get(value) || value}</option>
```
Comment (timoxley) [inline app/src/editor/canvas/components/Ports/Value/Select/index.jsx]: should probably handle case where `optionMap.get(value)` is `0`

File: app/src/editor/canvas/components/Ports/Value/Select/select.pcss [hunk 2]
```diff
@@ -19,8 +19,7 @@
   outline: 0;
   padding: 0 calc(0.5 * var(--um));
   transition: 200ms linear;
-  transition-property: background-color, color, width, box-shadow;
-  width: 100%;
+  transition-property: background-color, color, box-shadow;
```
Comment (timoxley) [inline app/src/editor/canvas/components/Ports/Value/Select/select.pcss]: ah. The solution was no width at all. 👏 👏

Comment (mondoreale) [reply app/src/editor/canvas/components/Ports/Value/Select/select.pcss]: Yep, you don't wanna `width: 100%` on the invisible `select`.

File: app/src/editor/canvas/components/Ports/Value/Select/index.jsx (hunk 1 above)
Comment (mondoreale) [reply app/src/editor/canvas/components/Ports/Value/Select/index.jsx]: Done.

**Pull request merged**